spec/ddtrace/profiling/ext/cthread_spec.rb

```diff
@@ -283,7 +283,7 @@ def on_main_thread
           end
 
           with_profiling_extensions_in_fork(
-            fork_expectations: proc do |status, stderr|
+            fork_expectations: proc do |status:, stdout:, stderr:|
```

Comment:
Minor: We can also use use `**_` here to ignore `stdout`.

Reply:
Either way looks fine I guess 🤷